index.d.ts

```diff
@@ -34,18 +34,44 @@ declare namespace pug {
   export function compileFileClient(path: string, options?: Options): string;
 
   /**
+   * Compile a Pug template and render it without locals to html string.
+   */
+  export function render(template: string): string;
+  /**
    * Compile a Pug template and render it with locals to html string.
+   * @param {(Options & LocalsObject)} options Pug Options and rendering locals
```

Comment:
There's 1000 other definitions where this isn't used. I don't think it's a valuable addition, as anyone using the tooling would understand how this works today, and the future improvements from TypeScript will also be a welcome addition.

However, the issue here is also that the types need to be removed from JSDoc.